### PR TITLE
Add promo bar

### PR DESCRIPTION
**Pull request by kasperbirch1**
#### Add logic to find a reservation that is available before the selected

A new useAlternativeAvailableManifestation hook has been added. It receives a work and returns an alternative manifest with few or 0 reservations. 

if needed it can also just return a boolean isOtherManifestationPreferred.

it is used in the reservation modal under edition.  

#### PromoBar is a new reusable component 
PromoBar accepts the following props:
- type: decide which icon to use
- text: is the text that is displayed
- theme: decide which text and background color
- sticky: sets a negative margin that matches the gap used in .reservation-modal-list

The first place it should be used is together with reservation modal items to show information about faster reservation

#### Screenshot of the result
Skærmbillede 2022-12-02 kl  13 10 59 (3)
Skærmbillede 2022-12-02 kl  13 11 13 (3)

If your change affects the user interface you should include a screenshot of the result with the pull request.

#### Checklist

- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [x] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

### CODE REVIEW

File: src/apps/material/material.dev.tsx [hunk 1]
```diff
@@ -536,6 +536,17 @@ export default {
...
       control: { type: "text" }
+    },
+    skipQueueMaterialIsAvailableInAnotherEditionText: {
+      name: "Skip queue material is available in another edition",
+      defaultValue:
+        "Skip the queue - The material is available in another edition",
+      control: { type: "text" }
+    },
+    skipQueueMaterialIsAvailableInAnotherEditionReservationsText: {
+      name: "Skip queue reservations cont",
```
Comment (spaceo) [inline src/apps/material/material.dev.tsx:547]: Do not understand name. What does it mean?

Comment (kasperbirch1) [reply src/apps/material/material.dev.tsx:617]: I have changed it to materialIsAvailableInAnotherEditionReservationsText 😊

File: src/components/reservation/ReservationModalBody.tsx [hunk 2]
```diff
@@ -196,7 +207,20 @@ const ReservationModalBody = ({
...
+                  classNames="px-35"
+                  sticky
+                  type="info"
+                  text={`${t("materialIsAvailableInAnotherEditionText")}
+                    - ${otherManifestationPreferred.titles.main}
+                    - ${getAuthorLine(otherManifestationPreferred, t)}
+                    - ${t(
+                      "materialIsAvailableInAnotherEditionReservationsText"
+                    )}:
+                    ${otherManifestationPreferred.reservations}`}
```
Comment (kasperg) [inline src/components/reservation/ReservationModalBody.tsx:221]: Please update this to use placeholders and plural handling.

**Comment by spaceo** [comment]
LGTM 👍

**Pull request merged**